parsec/class/parsec_object.h

```diff
@@ -470,6 +480,7 @@ static inline parsec_object_t *parsec_obj_new(parsec_class_t * cls)
     if (NULL != object) {
         object->obj_class = cls;
         object->obj_reference_count = 1;
+        object->obj_release = &parsec_obj_destruct_and_free;
```

Comment:
Won't the run_constructor overwrite this with `parsec_obj_destruct` (w/o free)?

Reply:
The `parsec_obect_t` has no dedicated constructor, it's all done here.